autopilot/autopilot.go

```diff
@@ -8,24 +8,53 @@ import (
 	"go.sia.tech/renterd/bus"
 	"go.sia.tech/renterd/hostdb"
 	"go.sia.tech/renterd/internal/consensus"
+	rhpv2 "go.sia.tech/renterd/rhp/v2"
+	"go.sia.tech/renterd/wallet"
 	"go.sia.tech/renterd/worker"
+	"go.sia.tech/siad/types"
 )
 
 type Store interface {
 	Tip() consensus.ChainIndex
+	Synced() bool // TODO: should be added to `Tip` | should be moved over to Bus
+
 	Config() Config
 	SetConfig(c Config) error
 }
 
+// TODO: should be defined in bus package, should be the interface of the client
```

Comment:
why would the `Bus` interface be defined in the `bus` package?

Reply:
figured we could combine an interface here with the interfaces from the `bus` package but that does not make sense indeed